### DISCUSSION TITLE
How to proxy a specific route?

### DISCUSSION THREAD

**Discussion by gopeter** [architect/architect#1307] [Q&A]
Looking at the [docs](https://arc.codes/docs/en/reference/project-manifest/proxy), there seems to be no way to proxy just one specific route. I try to express my idea with some pseudo code:

```
@http
/api
  method get
  src proxy
  
/*
  method any
  src server

@proxy
testing http://localhost:4000
staging https://qa.example.biz
production https://example.biz
```

Would be great to have multiple proxies too, but this one would be good enough for a start 😄

**Comment by samirrayani** [comment]
@gopeter did you ever figure this out?
